x/stake/keeper_keys.go

```diff
@@ -16,13 +16,14 @@ var (
 	ParamKey               = []byte{0x00} // key for parameters relating to staking
 	PoolKey                = []byte{0x01} // key for the staking pools
 	ValidatorsKey          = []byte{0x02} // prefix for each key to a validator
-	ValidatorsBondedKey    = []byte{0x03} // prefix for each key to bonded/actively validating validators
-	ValidatorsByPowerKey   = []byte{0x04} // prefix for each key to a validator sorted by power
-	ValidatorCliffKey      = []byte{0x05} // key for block-local tx index
-	ValidatorPowerCliffKey = []byte{0x06} // key for block-local tx index
-	TendermintUpdatesKey   = []byte{0x07} // prefix for each key to a validator which is being updated
-	DelegationKey          = []byte{0x08} // prefix for each key to a delegator's bond
-	IntraTxCounterKey      = []byte{0x09} // key for block-local tx index
+	ValidatorsByPubKeyKey  = []byte{0x03} // prefix for each key to a validator by pubkey
+	ValidatorsBondedKey    = []byte{0x04} // prefix for each key to bonded/actively validating validators
+	ValidatorsByPowerKey   = []byte{0x05} // prefix for each key to a validator sorted by power
```

Comment:
Maybe for these index keys we should add the `Index` somewhere in the name? - to be more explicit, what you think?

Reply:
"PubKeyKey" <- is ugly

Reply:
Now `PubKeyIndexKey`.